models/webhooks/store_product_updated.yml

```
@@ -15,6 +15,15 @@ properties:
             type: string
           id:
             type: integer
+          attributes:
+            type: string

```

should be array of strings
```suggestion
            type: array

```

---

also we might need to mentioned the exact set of allowed values here, like `name`, `description`, etc